subprojects/logging/src/main/java/org/gradle/api/logging/configuration/WarningType.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.logging.configuration;
+
+import org.gradle.api.Incubating;
+
+/**
+ * Specifies the warning type a user wants to see
+ *
+ * @since 4.5
+ */
+@Incubating
+public enum WarningType {
```

Comment:
:x: Should be covered by a unit test.

Reply:
Thanks, now I think UT is no longer needed.